modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -219,6 +224,23 @@ instead.
 
 ---
 
+## Removals
+
+### Status Indicator model
+
+**PR:** [#2472](https://github.com/Workday/canvas-kit/pull/2700)
+
+`StatusIndicator` model functionality has been updated and the next hooks has been removed:
+
+- `useStatusIndicatorModel`
+- `useStatusIndicator`
+- `statusIndicatorColors`
+- `useStatusIndicatorIcon`
+
+There is no external usage found, so it's not breaking change.
+
+---
```

Comment:
Worded this a little different, thoughts?

```
As part of the refactor, we've removed the following exports that were primarily used to style the component:
- `useStatusIndicatorModel`
- `useStatusIndicator`
- `statusIndicatorColors`
- `useStatusIndicatorIcon`.

Since there was no usage of these by our consumers, there should be no breaking changes.
```

Reply:
put it as suggestion